src/platform/site-wide/sass/merger.scss

```diff
@@ -199,6 +194,24 @@
           padding: 0;
         }
       }
+      .fa {
+        background-color: $color-white;
+        &.fa-medkit {
+          color: $indigo-cool-60;
+        }
+        &.fa-book {
+          color:  $mint-cool-50;
+        }
+         &.fa-handshake-o {
+          color: $red-60;
+        }
+        &.fa-money {
+          color: $green-cool-50;
+        }
+        &.fa-id-card {
+          color: $violet-warm-60;
+        }
```

Comment:
I think the icon styles in here need some refactoring in general. They collide with all the other icons on our site and are generally a bit hard to understand in here. I don't think we need to set colors directly on the icon specific class at all. That couples colors to icons, when the could easily change independently later.

We probably don't need to fix them in this PR, but I think we should do it soon. We're going to have two different styles of icons on the homepage after one of the other tickets, one where the icons are white and one where the icons are the hub page color, depending on if they're in the primary boxes or links in the next section.

Going forward, I think we at least want to have color variables named for each hub area (in addition to the ones in here) and a set of classes like:

`hub-icon-primary`
`hub-icon-link`
`hub-color-disability` (and so on for each area)

And the markup would be something like `class="hub-primary-icon hub-color-disability fa fa-handshake-o"`

Alternatively, we could use the FA mixins and create classes (like `hub-icon-disability`) for each area that incorporate the particular FA icon and avoid having `fa fa-handshake-o` in our markup.

Reply:
That makes perfect sense to me. I was at first thinking utility classes like `.text-primary` to override solely the color of that element, but I couldn't think of a lot of use cases for that outside of this, so it might encourage bad practice.

Reply:
I think it would actually be a good idea to get that implemented as part of this PR. @RobAWilkinson what are your thoughts?

Reply:
Yeah makes perfect sense to add them as classes. I'm also a huge fan of having lots of little  classes  that you string together. This would shake out to something like this: 
`class ="violet-bg white fa fa-handshake"`

I think now we're going for somewhat of a modified BEM methodology?